opus/application/static_media/js/browse.js

```diff
@@ -1189,10 +1188,17 @@ var o_browse = {
         $(".dataTable thead tr").append("<th scope='col' class='sticky-header'></th>");
         $.each(columns, function(index, header) {
             let slug = slugs[index];
+
```

Comment:
I understand why you're doing this, but this really can't be the right way to handle this. It's too inflexible for future cases that might have things other than (Min) or (Max) that can be confused with units. Here's the right thing to do, even though it's a bit more complicated:

- Update results/views.py api_get_data_and_images().
  - In addition to the existing "columns" return field, there should be one called "columns_no_units".
  - Update the comment in the function header to indicate that columns returns units, and the new one doesn't.
  - After the line that says `labels = labels_for_slugs(cols_to_slug_list(cols))` add a new line that defines labels_no_units and passes the argument units=False into the labels_for_slugs function. See example code nearby.
  - Check both of them for None and throw an error (see next line after above).
  - In the definition of "data" below add the "columns_no_units" field in the obvious way.
- When you call initTable in loadData, pass in both columns and columns_no_units
- Change initTable to take two arguments.
- Store the second one in opus.col_labels_no_units
- Use the no-units version to initialize the data label
- Kill off all this stuff about (Min) and (Max)
- You'll have to change the metadata selector behavior to pass in the second argument as well

Reply:
The logic is updated in the latest commit.